src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        begin();
+
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        while (!input.equals("bye")) {
+            execute(input);
+            input = scanner.nextLine(); //ready for next input
+        }
+
+        bye();
+    }
+
+    private static void begin() {
+        endCommand();
+
```

Comment:
This repo contains a trivial amount of code, I will do the backup review instead